src/feature/feature_utils.cairo

```diff
@@ -15,13 +16,21 @@ use satoru::data::data_store::{IDataStoreSafeDispatcher, IDataStoreSafeDispatche
 /// # Returns
 /// whether the feature is disabled.
 fn is_feature_disabled(data_store: IDataStoreSafeDispatcher, key: felt252) -> bool {
```

Comment:
maybe this function should return `Result<bool, Array<felt252>>` like discussed on tg, and to match other modules

Reply:
For now im fixing as expected from GMX source, If this is gonna be decided I can change the return type 👍